python/python-core/populate-a-dictionary-exercise/populate-a-dictionary-exercise.md

```diff
@@ -27,17 +27,15 @@ setupCode:
 
 ## Content
 
-> 👩‍💻 Your task is to populate a `dictionary` using these two `lists`:
-
-```python
-keys = ['Name', 'Surname', 'Profession']
-values = ["Alan", "Turing", "Mathematician"]
-```
-
-Don't forget to output the new `dictionary`.
+> 👩‍💻 Your task is to:
+> - **create a dictionary with at least one key-value pair**
+> - **save the values in a different variable**
+> - **save the keys in a different variable**
+> - **clear the dictionary**
```

Comment:
how could we make it clear that they should use dictionary methods for these three?

maybe something like this?

```suggestion
> 👩‍💻 Using dictionary methods, your task is to:
> - **create a dictionary with at least one key-value pair**
> - **save the values in a different variable**
> - **save the keys in a different variable**
> - **clear the dictionary**
```

Reply:
Yeah, that sounds better